src/init_game.cpp

```diff
@@ -823,11 +823,6 @@ void deinitGame()
 		cpp_Free_CSteamID(currentLobby); //TODO: Remove these bodges.
 		currentLobby = NULL;
 	}
-	if ( lobbyToConnectTo )
-	{
-		cpp_Free_CSteamID(lobbyToConnectTo);
-		lobbyToConnectTo = NULL;
-	}
```

Comment:
To verify what happens when lobby kept open

Reply:
This variable does not exist anymore.
Even when it did it was just a steam id to the lobby we were invited to, stored for some asynchronous logic.
It is completely gone now.

Reply:
Oh thought this was current lobby, thought I saw that not being closed anymore